tests/sndrcv_tls_ossl_certless_clientonly.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+# alorbach, 2019-01-16
+# testing sending and receiving via OpenSSL TLS with anon auth using bare ipv4, no SNI
+# This file is part of the rsyslog project, released  under ASL 2.0
+
+# uncomment for debugging support:
+. ${srcdir:=.}/diag.sh init
+export NUMMESSAGES=25000
+# start up the instances
+#export RSYSLOG_DEBUG="debug nostdout noprintmutexaction"
```

Comment:
in general, the comment lines here and in other tests would better be in proper order (albeit we can guess what is meant).

Reply:
Copy & paset from original test